tests/wpt/mozilla/tests/css/ol_japanese_iroha_ref.html

```diff
@@ -20,10 +18,10 @@
 </head>
 <body>
 <ol>
-    <li>&#x3044;. Gryffindor</li>
```

Comment:
I changed it because the new font was causing the reftest to fail. After this change it no longer fails. I'm guessing the presence of the `.` is font- and browser-specific. In Firefox it shows a "Japanese comma" instead of a period.

Servo with new font:
![screenshot from 2016-06-08 10-06-08](https://cloud.githubusercontent.com/assets/87961/15903451/b186574a-2d61-11e6-9230-e631f5108652.png)

Servo with old font:
![screenshot from 2016-06-08 10-10-20](https://cloud.githubusercontent.com/assets/87961/15903456/baad3ac8-2d61-11e6-945a-8b4c68d6cfa7.png)


Firefox with new font:
![screenshot from 2016-06-08 09-36-22](https://cloud.githubusercontent.com/assets/87961/15902444/cc7c2a8e-2d5c-11e6-9970-c3a100322434.png)

Firefox with old font:
![firefox-old-font](https://cloud.githubusercontent.com/assets/87961/15903439/a652c8f4-2d61-11e6-898f-2e4aa92c4660.png)